tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -1840,7 +1847,7 @@
               <xsl:text>,"x-sap-precision":</xsl:text>
               <xsl:value-of select="$target/@Precision" />
             </xsl:if>
-            <xsl:if test="number($target/@Scale)=$target/@Scale">
+            <xsl:if test="$target/@Scale!='floating' and $target/@Scale!='variable' and number($target/@Scale)=$target/@Scale">

```

Comment:
Not sure if this is necessary, but my [XSLT processor](https://www.npmjs.com/package/xslt3) fails otherwise. Will raise in issue with https://saxonica.plan.io/.